src/redux/books/booksSlice.js

```diff
@@ -1,18 +1,36 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-const initialState = {
-  books: [],
-};
+const initialState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
 
 const booksSlice = createSlice({
   name: 'books',
   initialState,
   reducers: {
     addBook: (state, action) => {
-      state.books.push(action.payload);
+      state.push(action.payload);
```

Comment:
[OPTIONAL]
- [ ] Great job implementing the action to add a new book, everything works fine :+1: However, I'd like to highlight one point for your consideration. While your approach using `push` to add a new book to the state does work, it's worth noting that this method is not recommended in React. React encourages immutability, and directly mutating the state, as `push` does, can sometimes lead to unexpected behavior and make it harder to track changes. Kindly consider using in the future alternative methods, like creating a new array with the updated data using the spread operator (`state.books = [...state.books, action.payload]`). You can read more about React immutability [here](https://stackoverflow.com/questions/34385243/why-is-immutability-so-important-or-needed-in-javascript)